src/components/calcite-shell-panel/calcite-shell-panel.scss

```diff
@@ -27,3 +24,10 @@ $shell-panel-max-width: 360px !default;
 .content[hidden] {
   display: none;
 }
+
+:host([layout="leading"]) slot[name="action-bar"]::slotted(calcite-action-bar) {
```

Comment:
Can you verify this is working in edge. just curious.

Reply:
@driskull I'll have to hand that testing over to someone who has an Edge device. 😬

Reply:
TBH, it's a pretty minor styling update and could either be removed or allowed to not work on Edge.

Reply:
if this selector isn't understood by Edge, I'd expect the border to be absent. If that's ok then 👍 .